src/Microsoft.DotNet.Build.CloudTestTasks/CopyBlobsToLatest.cs

```diff
@@ -0,0 +1,231 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    public class CopyBlobsToLatest : AzureConnectionStringBuildTask
+    {
+        [Required]
+        public string ContainerName { get; set; }
+
+        [Required]
+        public string Product { get; set; }
+
+        [Required]
+        public string ProductVersion { get; set; }
+
+        [Required]
+        public string Channel { get; set; }
+
+        [Required]
+        public string Commit { get; set; }
+
+        public bool Coherent { get; set; }
```

Comment:
@livarcocc @nguerrera - Do we still need/want this `Coherent` terminology?  I assume with the latest product construction, every build should be "coherent".

Reply:
I think you're right. In this context, is this always a prodcon build?

Reply:
At the moment, yes, but the goal is to use this for Core-Setup and *eventually* CLI builds, too--including isolated builds.

So far I was planning on setting Coherent to false for Core-Setup and true for CLI for all prodcon builds.

If the term is going away for all 2.1+ builds, though, makes sense to remove it (and never publish the "latest coherent version" file).